docs/workflow.rst

```diff
@@ -379,8 +280,8 @@ are serial on the core).
 
   .. code-block:: console
 
-     $ ./beast/tools/setup_batch_beast_fit.py projectname datafile.fits \
-       --num_percore 2 --nice 19
+     $ beast.tools.setup_batch_beast_fit.py --num_percore 2 --nice 19 \
```

Comment:
Need to covert commands to command line versions (see prev comments).

Reply:
resolved per discussion below